packages/google_maps_flutter/lib/src/controller.dart

```diff
@@ -236,4 +236,15 @@ class GoogleMapController {
         'map#getLatLng', screenCoordinate._toJson());
     return LatLng(latLng[0], latLng[1]);
   }
+
+  /// Returns [double] corresponding to the distance in meters of the [LatLng] in the current map view.
+  ///
+  /// Return NaN (Not-a-Number) if unable to get distance in meters.
+  /// This is only accurate for small Earth distances.
+  Future<double> getDistance(double meters, LatLng latLng) async {
+    final double distance = await channel.invokeMethod<double>(
+        'map#getDistance',
+        <String, dynamic>{"meter": meters, "location": latLng._toJson()});
+    return distance;
```

Comment:
This should become a method in the `platform_interface` of the map, [here](https://github.com/flutter/plugins/blob/master/packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/platform_interface/google_maps_flutter_platform.dart#L235-L240) (and its [implementation](https://github.com/flutter/plugins/blob/master/packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/method_channel/method_channel_google_maps_flutter.dart#L432-L438)).